### PR TITLE
fix(website): Add "read blog" tag above Hero title, adjust hero margins and font sizes

### PR DESCRIPTION
**Pull request by Patticatti**

### CODE REVIEW

File: website/public/images/play-icon.png
Comment (jamilbk) [inline website/public/images/play-icon.png:1]: Can we use a smaller image or just a react-icon here? 12.5KB seems a lot

File: website/src/app/page.tsx [hunk 1]
```diff
@@ -48,6 +48,18 @@ export default function Page() {
     <>
       <section className="bg-neutral-950 bg-hero pt-28 mb-16">
         <div className="flex flex-col items-center mx-auto md:px-0 px-4 max-w-screen-md">
+          <Link href="/blog">
```
Comment (jamilbk) [inline website/src/app/page.tsx]: Could you make this a component so we can easily update the text and link for this?

**Review by jamilbk — CHANGES_REQUESTED** [review]
Screenshot 2024-09-03 at 9 29 08 AM

I think there's a slight black rounded background on the link? Can that be removed?

File: website/src/components/Banner/index.tsx [hunk 2]
```diff
@@ -1,23 +1,18 @@
...
-export default function Banner({
-  active,
-  overlay,
-  children,
-}: {
-  active: boolean;
-  overlay?: boolean;
-  children: React.ReactNode;
-}) {
-  if (!active) return null;
```
Comment (jamilbk) [inline website/src/components/Banner/index.tsx:13]: If you remove this check then the Banner will be unconditionally rendered on all pages.

File: website/src/app/changelog/layout.tsx [hunk 3]
```diff
@@ -2,7 +2,7 @@ import Image from "next/image";
 
 export default function Layout({ children }: { children: React.ReactNode }) {
   return (
-    <div className="pt-14 flex flex-col">
```
Comment (jamilbk) [inline website/src/app/changelog/layout.tsx]: @Patticatti I think this change needs to be reverted

**Comment by Patticatti** [comment]
> Screenshot 2024-09-03 at 9 29 08 AM
> I think there's a slight black rounded background on the link? Can that be removed?

wait wdym rounded background? i think i added it on purpose to give it more contrast

File: website/src/app/page.tsx [hunk 4]
```diff
@@ -39,6 +40,17 @@ export default function Page() {
     <>
       <section className="bg-neutral-950 bg-hero pt-28 mb-16">
         <div className="flex flex-col items-center mx-auto md:px-0 px-4 max-w-screen-md">
+          <Banner href="/blog" active={true}>
```
Comment (jamilbk) [inline website/src/app/page.tsx]:
```suggestion
          <Banner href="/blog" active>
```

**Review by jamilbk — CHANGES_REQUESTED** [review]

File: website/src/app/page.tsx [hunk 5]
```diff
@@ -39,6 +40,17 @@ export default function Page() {
...
         <div className="flex flex-col items-center mx-auto md:px-0 px-4 max-w-screen-md">
+          <Banner href="/blog" active>
+            <div className="hover:text-primary-450 transition-all duration-50 ease-in-out invert-0 flex gap-2 mb-5 items-center rounded-full p-1.5 text-neutral-400 bg-[rgba(255,255,255,0.02)] shadow-[inset_0_-8px_11px_0_rgba(255,255,255,0.05)]">
+              <Image
+                src="/images/play-icon.png"
+                width="24"
+                height="24"
+                alt="Play Icon"
+              />
+              <p className="text-sm mr-4">See our latest news</p>
```
Comment (jamilbk) [inline website/src/app/page.tsx]:
```suggestion
              <p className="text-sm mr-4">New: Internet Resources, REST API, and improved wildcard matching</p>
```

File: website/src/app/page.tsx [hunk 6]
```diff
@@ -39,6 +40,17 @@ export default function Page() {
     <>
       <section className="bg-neutral-950 bg-hero pt-28 mb-16">
         <div className="flex flex-col items-center mx-auto md:px-0 px-4 max-w-screen-md">
+          <Banner href="/blog" active>
```
Comment (jamilbk) [inline website/src/app/page.tsx]:
```suggestion
          <Banner href="/blog/sep-2024-update" active>
```

File: website/src/app/page.tsx (hunk 5 above)
Comment (jamilbk) [inline website/src/app/page.tsx]: Shouldn't all of this be in the `Banner` component so we don't need to duplicate it in `children`?

**Review by jamilbk — CHANGES_REQUESTED** [review]

File: website/src/app/page.tsx (hunk 5 above)
Comment (Patticatti) [reply website/src/app/page.tsx]: Yes lemme fix that

**Review by jamilbk — APPROVED** [review]
Updated the banner style

**Pull request merged**